src/components/segmentedControl/segment.tsx

```diff
@@ -49,6 +49,10 @@ export type SegmentProps = SegmentedControlItemProps & {
    * onLayout function.
    */
   onLayout?: (index: number, event: LayoutChangeEvent) => void;
+  /**
+   * Additional style for the segment.
+   */
+  segmentStyle?: StyleProp<ViewStyle>;
```

Comment:
Why this is not called simply "style" as it is the Segment component? 
I see in the API file that there is a "style" prop but I don't see that it is used anywhere...

Reply:
Renamed to `style` inside `Segment`.
The `style` prop in the API file is for the whole component.